src/create-config.js

```diff
@@ -0,0 +1,23 @@
+import defaultRTSuffixes from './request-type-suffixes';
+
+let config = {
+  requestSuffixes: defaultRTSuffixes
+};
+
+export const saveConfig = newConfig => {
```

Comment:
this generating of config is weird, @mieszko4 do you have some idea how to make it nicer?

Reply:
this is ok, since module is loaded only once. but maybe naming could be improved.
I would do:
```es6
export const setRequestSuffixes(requested, completed, failed) {
  config = {
    ...config,
    requestSuffixes: [requested, completed, failed]
  };
}
```

Other solution could be setting that up in `package.json` and reading the config from there.